tests/io/datasets/test_dm.py

```diff
@@ -1,15 +1,16 @@
 import os
-import glob
+from glob import glob
```

Comment:
This breaks `test_dm_dist` - please have a look at [the docs](https://libertem.github.io/LiberTEM/dist_tests.html) to see how to run these tests, which should reproduce the problem.